cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/vsphere/vsphere-mc-common/VsphereCredentialsStep.tsx

```diff
@@ -218,7 +218,7 @@ export function VsphereCredentialsStep(props: Partial<StepProps>) {
         } else {
             setLoadingDatacenters(false);
         }
-    }, [connected]);
+    }, [connected, clearDatacenters]);
```

Comment:
This change would not work anymore. Be careful of changes to files you're not working on; I'm guessing this occurred "automatically" via prettier?

Reply:
This wasnt me it was part of my rebase with the tce-ui-poc branch since I was having issues

Reply:
It probably was due to prettier.